src/features/typeDecorations.ts

```diff
@@ -12,6 +14,28 @@ export default () => {
         },
         rangeBehavior: vscode.DecorationRangeBehavior.ClosedClosed,
     })
+
+    const checkIfInStyles = async (document: vscode.TextDocument, position: vscode.Position) => {
+        if (!getExtensionSetting('typeDecorationsInStyles.enable')) return false
```

Comment:
I understand the solution insn't efficient, but right now I just wanted it to work :)